FirebaseSharedSwift/Sources/third_party/FirebaseDataEncoder/FirebaseDataEncoder.swift

```diff
@@ -1251,8 +1251,8 @@ fileprivate class __JSONDecoder : Decoder {
                                         DecodingError.Context(codingPath: self.codingPath,
                                                               debugDescription: "Cannot get keyed decoding container -- found null value instead."))
     }
-
-    guard let topContainer = self.storage.topContainer as? [String : Any] else {
```

Comment:
@mortenbekditlevsen or anyone: Any suggestions about ways to decode RemoteConfigValues without modifying this file?

Reply:
Hmm.. if possible I think it'd be reasonable to have a `RemoteConfigEncoder/Decoder` that has this logic and then defers to this decoder under the hood. I haven't thought much about it so it could be a silly suggestion, but I agree modifying this file and having RC specific code in it should be avoided if possible.